### PR TITLE
[DOC] Draft decision-making file

### LINKED ISSUE #526: Define and document decision-making procedure
### Summary
I think it would be a good idea to set up some criteria for making decisions on proposed changes to the repository (especially those with the [discussion label](https://github.com/ME-ICA/tedana/labels/discussion).

This is related to #515, as decision-making and code review are related tasks.

### Additional Detail
This ties into the ongoing governance discussion, but I think is distinct from it. While we do currently have a BDFL, there are plenty of more minor decisions that can be made via vote (e.g., #487, #524). However, we don't have any criteria for making the final decision based on votes (e.g., time to wait before closing the "poll", minimum number of votes needed to make a decision, etc.). BIDS has a [DECISION-MAKING.md](https://github.com/bids-standard/bids-specification/blob/ad7a16c9bf6eeaa9abe76236f1a2f185fc71daa1/DECISION-MAKING.md) file in the specification repository that might be a useful template.

### Next Steps
1. Define decision-making procedure
    1. What constitutes a voteable issue vs. one that requires a decision specifically from leadership?
    2. How many votes are required to make a decision?
    3. How long should voteable issues remain open before a decision is made or the poll is closed?
2. Incorporate new decision-making procedure into documentation.

### PR DESCRIPTION
**Pull request by tsalo**
Closes #526.

Changes proposed in this pull request:

- Create DECISION-MAKING.md, copying many elements from the BIDS standard's equivalent file.

### CODE REVIEW

File: DECISION-MAKING.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+   1. Does not feature "WIP" in the title (Work in Progress).
+   1. Passes all automated tests.
+   1. Is not proposing a new release or has been approved by at least one
+      Maintainer (i.e., PRs proposing new releases need to be approved by at
+      least one Maintainer).
+1. The Project Leader can merge any PR - even if it's not eligible to merge
+   according to Rule 4.
+1. Any Maintainer can Review a PR and request changes. If a Maintainer
+   Requests changes they need to provide an explanation regarding what changes
+   should be added and justification of their importance. Reviews requesting
```
Comment (jbteves) [inline DECISION-MAKING.md]: This is a really good point, thanks for making sure it's there.

**Comment by tsalo** [comment]
@dowdlelt 
> While I (kinda) love being at the top of the list of maintainers (and I am really hoping to live up to that position...in time) I truly dislike that it is just due to the tyranny of the alphabet rather than effort. Should it be first sorted by time, then alphabet? I'm not strongly opposed mind you, I just think it unfairly favors me (my name is early here, on posters, on github, etc)

I figured that folks might fluctuate in their personal time commitments, so I was reticent to sort by time (it will make the diffs for PRs changing time commitments messier). That said, I'm not married to the alphabetical sorting, but I'm also not concerned about the order personally.

@jbteves 
> Thanks so much for writing this up, @tsalo ! I know it's very difficult to express all of these ideas in writing so clearly.

Thank you, although it's almost all just copied over and lightly adapted from the BIDS specification.

> How do we want to approve this as our file?

I think we should probably discuss it on the next call, rather than using the approach proposed in the file to vote on the file itself.

**Comment by jbteves** [comment]
I'm sorry, I completely missed your response @tsalo.

RE: alphabetical order. I agree that it's an arbitrary choice, but I think trying to measure involvement with the project is hard, and costs effort that would better be allocated elsewhere. If it makes you feel better, we could be unique and do reverse-alphabetical, which is an equally arbitrary ordering :- )

RE: spec. It's still work, and I appreciate it. The projects are in many ways quite different.

RE: approval. I agree, a discussion on the call would be good. Hopefully despite the pandemic most will be able to make it.

**Comment by tsalo** [comment]
Per the April devs call, we may want to wait on @KirstieJane's input regarding triage before merging this PR.

**Comment by tsalo** [comment]
I'm going to pause this, as we're moving our development of the decision-making file to a Google Doc for the foreseeable future. I will copy over the changes to this PR once they're finalized in the Doc.

**Comment by handwerkerd** [comment]
In response to the last dev call and #607, I'm going to start drafting a PR to put all the info in the [decision making document](https://docs.google.com/document/d/1213KX0Jsr-I0aVHIF3Xdp3EZhORMWEoSDR7rT948K8U/edit?usp=sharing) into the documentation. It's not always obvious where things should go so I wanted to run my plan by others. The first question is whether I should add onto this PR or should I open a new PR and we'll close this one? Thoughts @tsalo @emdupre @jbteves @eurunuela ?

Most of the current info is in the [Contributing to tedana](https://tedana.readthedocs.io/en/latest/contributing.html#) docs page. I'd remove [Governance](https://tedana.readthedocs.io/en/latest/contributing.html#governance) as a subheading on that page and make it a distinct page that will contain the info on the steering committee composition & role as well as the decision making procedure. This PR also made it a distinct PR, but didn't yet link it to the rest fo the documentation.

Remaining on the Contributing to tedana page will be the link to the practical contributors' guide and the [development philosophy](https://tedana.readthedocs.io/en/latest/contributing.html#tedana-s-development-philosophy). The tedana project scope from the governance doc will be added to the development philosophy section.

The other potential place to edit is the`Project Vision` on the [Roadmap](https://tedana.readthedocs.io/en/latest/roadmap.html) page. Some of the scope could be repeated there, but, in general, that vision should align more closely with the scope.

The last big part is updating the contributors section to be more descriptive. That effort could be included in the same PR or a separate PR. We talked about making a distinct PR for that during the last dev call and I think that does make sense.

**Comment by jbteves** [comment]
I'd vote new PR, but open to other suggestions.

We can make the contributing section expansion a different PR, I think that makes sense if we feel the other two are more or less stable.

**Comment by tsalo** [comment]
>In response to the last dev call and #607, I'm going to start drafting a PR to put all the info in the decision making document into the documentation. It's not always obvious where things should go so I wanted to run my plan by others. The first question is whether I should add onto this PR or should I open a new PR and we'll close this one? Thoughts @tsalo @emdupre @jbteves @eurunuela ?

I'm happy to close this PR given how out-of-date it is.

> We can make the contributing section expansion a different PR, I think that makes sense if we feel the other two are more or less stable.

Agreed. A separate PR sounds good.

I don't want to close this PR until this discussion is complete, though. @handwerkerd When you're happy with the state of that discussion, would you mind closing this PR?

**Comment by handwerkerd** [comment]
OK. When I open the new draft PR, I'll close this one.

**Comment by eurunuela** [comment]
I agree with what @jbteves and @tsalo suggested. 

I'd also make a PR with the updates to the contributors section.

**Comment by handwerkerd** [comment]
I opened #615 so I'm closing this PR.

**Pull request closed without merging**